dora/src/cannon/codegen.rs

```diff
@@ -904,6 +904,31 @@ where
             .store_mem(bytecode_type.mode(), Mem::Local(offset), reg);
     }
 
+    fn emit_store_global_field(&mut self, src: Register, global_id: GlobalId) {
```

Comment:
Why `field`? Could you please rename this.

Reply:
I copied the `emit_load_global_field` method header and just renamed it. Renamed both methods and removed the `field` part of the name.